EtageControl/test/verify/entry.js

```diff
@@ -0,0 +1,96 @@
+// Run: npm run test-web
```

Comment:
can this be moved to shared code at the top level of the repo?

Reply:
Yes, but needs some work/troubleshooting beforehand - of particular note is L96 of this file; it will need to change for each plugin

Reply:
Understood @rionastokes . Thanks for the review. Understood @rjrodger , I'll do that.

Reply:
Noted - this would mean we can keep the convention of running tests per plugin. Might look at adding a "run all plugins in the repo" test in the future